dev-embedded/arduino/arduino-1.8.7.ebuild

```diff
@@ -0,0 +1,146 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit java-pkg-2 java-ant-2 gnome2-utils
+
+DESCRIPTION="An open-source AVR electronics prototyping platform"
+HOMEPAGE="https://arduino.cc/ https://github.com/arduino/"
+
+ARDUINO_DOCS=(
+	"reference-1.6.6-3"
+	"Galileo_help_files-1.6.2"
+	"Edison_help_files-1.6.2"
+)
+
+for docname in "${ARDUINO_DOCS[@]}"; do
+	ARDUINO_DOCS_URI+=" https://downloads.arduino.cc/${docname}.zip -> ${PN}-${docname}.zip"
```

Comment:
Wouldn't it make more sense to add here ${P} instead of PN ?

Reply:
So you think there will not be any conflicts in tarballs ?

Reply:
Should not. Because doc file still has version in it's name (arduino-reference-1.6.6-3.zip). Let's say they will update docs in next version, so there will be arduino-1.8.7 with arduino-reference-1.6.6-3 and arduino-1.8.8 with arduino-reference-1.6.6-4. I see no conflicts here.